core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -71,8 +71,8 @@ beforeAll(() => {
         expect(jazzServices_po.getFunctionServiceName().getText()).toEqual(servicename);
         expect(jazzServices_po.getFunctionType().getText()).toEqual('function');
         expect(jazzServices_po.getFunctionStatus().getText()).toEqual('creation started');
-    });
-  it('Create Website Service', () => {
+  });
+it('Create Website Service', () => {
```

Comment:
fix indentation everywhere

Reply:
Yes

Reply:
well you shld give one tab to it block. it seems it is at same level as decribe?